[profiling] Fix missing instructions for Java custom context (#22229)
DataDog/documentation: content/en/profiler/guide/isolate-outliers-in-monolithic-services.md

}
```

> Then inform the Datadog backend of your intent to filter by this context key by setting the `profiling.context.attributes` configuration with one of:

Reviewer: ```suggestion
Then, inform the Datadog backend of your intent to filter by this context key; set the `profiling.context.attributes` configuration with one of the following:
```

* Environment variable: `DD_PROFILING_CONTEXT_ATTRIBUTES=customer_name`
* System setting: `-Ddd.profiling.context.attributes=customer_name`

> If you have several such context keys you should use a comma-separated string for the configuration (`-Ddd.profiling.context.attributes=customer_name,customer_group` for example).

Reviewer: ```suggestion
If you have multiple context keys, use a comma-separated string for the configuration (for example,`-Ddd.profiling.context.attributes=customer_name,customer_group`).
```
